.storybook/components/LinkComponent.tsx

```diff
@@ -0,0 +1,23 @@
+import { PropsWithChildren, forwardRef } from 'react';
```

Comment:
Honestly, I'd really like to move this into it's own package. We have duplicated this code 4 times now.

Will raise another PR shortly.